pages/contracts/erc6538-contract.mdx

```diff
@@ -0,0 +1,154 @@
+# ERC6538Registry.sol
+
+## Overview
+
+The `ERC6538Registry` contract is designed to map accounts to their stealth meta-addresses. This mapping is done according to the [ERC-6538](https://eips.ethereum.org/EIPS/eip-6538) standard. The contract provides functionalities to register, update, and verify stealth meta-addresses. Additionally, it ensures the security and integrity of these registrations through the use of nonces and signatures.
+
+### Code: [`ERC6538Registry.sol`](https://github.com/ScopeLift/stealth-address-erc-contracts/blob/b6f7d989b6247f81ceede5d1be2ffb23c1cb39b5/src/ERC6538Registry.sol)
+
+## Events
+
+### StealthMetaAddressSet
+
+The `StealthMetaAddressSet` event is emitted when a registrant updates their stealth meta-address.
+
+- **registrant** (`address`): The account that the registered the stealth meta-address belongs to.
+- **schemeId** (`uint256`): Identifier corresponding to the applied stealth address scheme (e.g., `1` for `secp256k1` as specified in [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564)).
+- **stealthMetaAddress** (`bytes`): The stealth meta-address. Based on [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564).
+
+#### Event Signature
+
+```solidity
+event StealthMetaAddressSet(
+  address indexed registrant, uint256 indexed schemeId, bytes stealthMetaAddress
+);
+```
+
+### NonceIncremented
```

Comment:
maybe events, errors, and funcs can be `code` to differentiate between other headers

Reply:
```suggestion
### `NonceIncremented`
```

Reply:
Fixed in https://github.com/ScopeLift/stealth-address-docs/pull/12/commits/f98ca208f1f49f9768386581c15f682ab4fc877a